### DISCUSSION TITLE
Change to Starter Code Repo Timeline????

### DISCUSSION THREAD

**Discussion by doctorscott** [community/community#86078] [GitHub Education] [answered]
### Hi everyone,

Hello! Just wondering when [this](https://github.blog/changelog/2023-12-08-upcoming-changes-to-assignment-repositories-and-starter-code-in-github-classroom/) change is being pushed out? We really need to know! The date "In January" is not sufficient. Please update ASAP or cancel this and do it next summer.

**Comment by RyanHecht** [comment, marked as answer]
Hi @doctorscott,

We're planning on rolling this out as a Public Beta that you may opt-in to on a per-Classroom basis very soon. The general rollout (when all Classrooms will be subject to the new system) will then be scheduled for the summer.

We're working on updating the changelog post with this information!
